roles/kubernetes-apps/dashboard/templates/kubernetes-dashboard.yml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Just a thought. I am not sure it's good idea to copy 'kubernetes-dashboard.yml' from [github](https://github.com/kubernetes/dashboard/blob/master/src/deploy/kubernetes-dashboard.yaml)

There can be some changes in upstream. Perhaps, we should just get it from github directly.